webpack/assets/javascripts/react_app/components/hosts/storage/vmware/index.js

```diff
@@ -93,10 +131,43 @@ class StorageContainer extends React.Component {
   }
 }
 
+StorageContainer.propTypes = {
+  addController: PropTypes.func,
+  addDisk: PropTypes.func,
+  config: PropTypes.object,
+  controllers: PropTypes.array,
+  data: PropTypes.object,
+  datastores: PropTypes.object,
```

Comment:
PropTypes.object is not informative 
`Proptype.shape` is much better for arrays and objects.
i.e:
```javascript
PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  }).isRequired, // if a prop is required, add `.isRequired`, else just add a default value
```